components/serial-libs/superlu/SPECS/superlu.spec

```diff
@@ -54,34 +50,34 @@ linear systems that the author is aware of.
 Docu can be found on http://www.netlib.org.
 
 %prep
-%setup -q -n SuperLU_%{version}
+%setup -q -n superlu-%{version}
 %patch -p1
 %patch1 -p1
-%patch2 -p1
-%patch3 -p1
 
 %build
 %ohpc_setup_compiler
-export DEFAULT_OPTS="${CFLAGS} -Wno-implicit-int"
-export DEFAULT_OPTS="${DEFAULT_OPTS} -Wno-implicit-function-declaration"
+export DEFAULT_OPTS="${CFLAGS} -Wno-implicit-int -Wno-implicit-function-declaration -fPIC -DPIC"
+export BLAS_LIB_EXPORT="-lopenblas"
+export USE_VENDOR_BLAS=1
 %if "%{compiler_family}" == "arm1"
 export DEFAULT_OPTS="${DEFAULT_OPTS} -fsimdmath"
 %endif
 
-make lib
+cmake -S . -B build -DCMAKE_INSTALL_PREFIX=./build -DCMAKE_C_FLAGS="${DEFAULT_OPTS}"
+cmake --build build --target superlu
 
-mkdir tmp
-(cd tmp; ar -x ../SRC/libsuperlu.a)
-$FC -shared -Wl,-soname,libsuperlu.so.4 -o lib/libsuperlu.so tmp/*.o
+mkdir tmp lib
+(cd tmp; ar -x ../build/SRC/libsuperlu.a)
+$FC -shared -Wl,-soname,libsuperlu.so.7 -o lib/libsuperlu.so tmp/*.o
 
 %install
 mkdir -p %{buildroot}%{install_path}/lib
 mkdir -p %{buildroot}%{install_path}/include
 install -m644 SRC/*.h %{buildroot}%{install_path}/include
 install -m755 lib/libsuperlu.so %{buildroot}%{install_path}/lib/libsuperlu.so.%{version}
 pushd %{buildroot}%{install_path}/lib
-ln -s libsuperlu.so.%{version} libsuperlu.so.4
-ln -s libsuperlu.so.4 libsuperlu.so
+ln -s libsuperlu.so.%{version} libsuperlu.so.7
```

Comment:
Why did you use 7? Is that used by upstream or are just following the package version?

Reply:
Following the package version. 
Should I revert to v4?